### PR TITLE
ci/carton test --environment defaultbrowser

### PR DESCRIPTION
**Pull request by thecb4**
Validating that the tests perform on GitHub Actions without additional steps before formal request.

### CODE REVIEW

**Comment by thecb4** [comment]
This pull request is in support of #99

File: Tests/CartonCommandTests/CommandTestHelper.swift [hunk 1]
```diff
@@ -246,6 +246,7 @@ public extension XCTest {
     command: String,
     cwd: URL? = nil, // To allow for testing of file based output
     expected: String? = nil,
+    expectedContains: Bool = false,
```
Comment (MaxDesiatov) [inline Tests/CartonCommandTests/CommandTestHelper.swift:249]: Can a doc comment be added for this function? It's not quite clear what `expectedContains` is supposed to indicate, and a doc comment could clarify that and also increase our documentation coverge in general.

File: Tests/CartonCommandTests/TestCommandTests.swift [hunk 2]
```diff
@@ -15,16 +15,26 @@
...
 
+import AsyncHTTPClient
 @testable import CartonCLI
 import TSCBasic
 import XCTest
 
 extension TestCommandTests: Testable {}
 
 final class TestCommandTests: XCTestCase {
+  var client: HTTPClient?
```
Comment (MaxDesiatov) [inline Tests/CartonCommandTests/TestCommandTests.swift:26]: nit

```suggestion
  private var client: HTTPClient?
```

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+version: '3.7'
```
Comment (MaxDesiatov) [inline docker-compose.yml:1]: If this file is supposed to be in the repository, can a clarification be added to `README.md` describing a proper way to use it?

Comment (thecb4) [reply docker-compose.yml:1]: Ran some additional tests to validate removing this won't be an issue. I will remove it.

File: Tests/CartonCommandTests/CommandTestHelper.swift (hunk 1 above)
Comment (thecb4) [reply Tests/CartonCommandTests/CommandTestHelper.swift:262]: Comments added

File: Tests/CartonCommandTests/TestCommandTests.swift (hunk 2 above)
Comment (thecb4) [reply Tests/CartonCommandTests/TestCommandTests.swift]: Scope modifier added

**Review by MaxDesiatov — APPROVED** [review]
LGTM, thank you! 🙌

**Pull request merged**